events/elections/2020/README.md

```diff
@@ -67,18 +82,18 @@ email saying something like "I accept the nomination".
 
 Once nominated, you must get the endorsement of three (3) different eligible
 voters from three (3) different employers.  If you are eligible to vote
-yourself, you count as one of the three. Endorsements from non-voting members does not count towards the final count. 
+yourself, you count as one of the three. Endorsements from non-voting members does not count towards the final count.
 
 [Eligible voters] may endorse candidates of their choosing by replying to the
-candidate's nomination email saying something like "I endorse this nominee"
-or "+1". Please specify your github ID and state that you are in voters.md.
+candidate's nomination email saying something like "I endorse this nominee, and I work for <COMPANY>"
+or "+1". Please specify your github ID, state that you are in voters.md, and include your employer's name so that we can which candidates have sufficient endorsements.
```

Comment:
nit: this probably misses a word..."so that we can **see** which candidates have sufficient endorsements?"

Reply:
I'll fix this after the initial version is merged -- at this point no time for further lgtms.  thanks!